### DISCUSSION TITLE
image not showing

### DISCUSSION THREAD

**Discussion by DaveMolla** [filamentphp/filament#3973] [Help]
This is the code for form schema :

SpatieMediaLibraryFileUpload::make('thumbnail')->collection('posts')
And this is the code for table columns

SpatieMediaLibraryImageColumn::make('thumbnail')->collection('posts')

I have inserted an image for the third post but its not showing
![Screenshot (280)](https://user-images.githubusercontent.com/92756138/189621408-08a7b2ee-d9f3-4235-a10f-2b9f6d34b04c.png)

**Comment by danharrin** [comment]
What is the console error? Check your APP_URL is correct

**Reply by DaveMolla** [reply]
Do you mean this?
![Screenshot (281)](https://user-images.githubusercontent.com/92756138/189631473-ff4f98ca-06b6-48d4-b80f-62a502e98f9c.png)

**Reply by DaveMolla** [reply]
And here is the APP_URL 
![Screenshot (282)](https://user-images.githubusercontent.com/92756138/189631861-4057380b-f4af-4903-a10b-b9c048373cd9.png)

**Reply by danharrin** [reply]
Your APP_URL should be `http://127.0.0.1:8000`

Otherwise, I don't know the problem. Something in your app is misconfigured
